src/genie/libs/parser/iosxe/show_ipv6_protocols.py

```diff
@@ -0,0 +1,979 @@
+                    }
+                }
+            },
+        }
+    }
+
+
+# ==============================
+# Parser for 'show ipv6 protocols'
+# ==============================
+class ShowIpv6Protocols(ShowIpv6ProtocolsSchema):
+
+    ''' Parser for "show ip protocols" '''
+
+    cli_command = [
+        'show ipv6 protocols',
+        'show ipv6 protocols vrf {vrf}'
+    ]
+
+    exclude = ['last_update', ' network', 'next_update']
+
+    def cli(self, vrf="", cmd="", output=None):
+        if output is None:
+            if not cmd:
+                if vrf:
+                    cmd = self.cli_command[1].format(vrf=vrf)
+                else:
+                    cmd = self.cli_command[0]
+            # get output from device
+            out = self.device.execute(cmd)
+        else:
+            out = output
+
+        # Init vars
+        ret_dict = {}
+
+        if not vrf:
+            vrf = "default"
+        protocol = None
+        # Routing for Networks:
+        routing_networks = False
+        routing_network_intfs = []
+        # Routing on Interfaces Configured Explicitly
+        routing_on_interfaces = False
+        routing_on_interfaces_intfs = []
+        # Routing Information Sources:
+        routing_information = False
+        routing_info_gateways = []
+        # Passive Interface(s):
+        passive_interfaces = False
+        passive_intfs = []
+
+        # Routing Protocol is "ospf 1"
+        # Routing Protocol is "bgp 100"
+        # Routing Protocol is "isis"
+        # Routing Protocol is "isis banana"
+        # Routing Protocol is "eigrp 1"
+        p1 = re.compile(r"^IPv6 Routing +Protocol +is"
+                        r" +\"(?P<protocol>(ospf|bgp|isis|eigrp|rip))"
+                        r"(?: *(?P<pid>(\S+)))?\"$")
+
+        # Outgoing update filter list for all interfaces is not set
+        # Incoming update filter list for all interfaces is not set
+        p2 = re.compile(r"^(?P<dir>(Outgoing|Incoming)) +update +filter +list"
+                        r" +for +all +interfaces +is +(?P<state>([a-zA-Z\s]+))$")
+
+        # Router ID 10.4.1.1
+        p3 = re.compile(r"^Router +ID +(?P<router_id>(\S+))$")
+
+        # Number of areas in this router is 1. 1 normal 0 stub 0 nssa
+        p4 = re.compile(r"^Number +of +areas:"
+                        r" +(?P<normal>(\d+)) +normal,"
+                        r" +(?P<stub>(\d+)) +stub, +(?P<nssa>(\d+)) +nssa$")
+
+        # EIGRP-IPv6 VR(pyats) Address-Family Protocol for AS(20)
+        # EIGRP-IPv4 Neighbors for AS(100)
+        # EIGRP-IPv4 Neighbors for AS(100) VRF(VRF1)
+        p5 = re.compile(
+            r'^EIGRP\-(?P<address_family>IPv4|IPv6)\s+(VR\((?P<name>\w+)\)\s+Address\-Family\s+)?'
+            'Protocol\s+for\s+AS\(\s*(?P<as_num>[\S]+)\)\s*(?:VRF\((?P<vrf>\S+)\))?$'
+        )
+
+        # Routing for Networks:
+        p6_1 = re.compile(r"^Routing +for +Networks:$")
+
+        # Interfaces Configured Explicitly (Area 0):
```

Comment:
This should probably be `Interfaces (Area 0):`
```suggestion
        # Interfaces (Area 0):
```